src/Meilisearch/Version.cs

```diff
@@ -0,0 +1,26 @@
+namespace Meilisearch
+{
+    /// <summary>
+    /// Information regarding an API key for the Meilisearch server.
+    /// </summary>
+    public class Version
+    {
+        /// <summary>
+        /// Extracts version from Meilisearch.csproj.
+        /// </summary>
+        /// <returns>Returns a formatted version.</returns>
+        public string GetQualifiedVersion()
+        {
+            return $"Meilisearch (v{this.GetVersion()})";
```

Comment:
Is this the package version or the version of Meilisearch?

Reply:
The idea is to retrieve the package version (in this case the dotnet package version) - currently 0.9.0

Reply:
It is not very clear that it is the version of the dot-net package and not the meilisearch engine. 
```suggestion
            return $"Meilisearch NET. (v{this.GetVersion()})";
```
maybe this? 
If you prefer not, no problem at all :) both are good

Reply:
Good catch!